modules/account-map/modules/roles-to-principals/variables.tf

```diff
@@ -17,8 +17,8 @@ variable "privileged" {
 
 variable "global_tenant_name" {
   type        = string
-  description = "Global tenant name"
-  default     = null
+  description = "The tenant name used for organization-wide resources"
+  default     = "mgmt"
```

Comment:
Why does this default to mgmt ?

Reply:
Otherwise we'd always have to supply this value when instantiating the submodule. But we can make it `null` if you want.

Reply:
Because this is a sub-module, we cannot supply a `default.auto.tfvars`, and we do not want to have to chain this through every client module, so in general this value should be directly edited/changed by the user here and in `../iam-roles/variables.tf`.